src/components/auth/signUp/SignUp.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { auth } from "../../config/firebaseConfig";
+import db from "../../config/firebaseConfig";
+
+export default function SignUp() {
+  const createNewUser = async (e) => {
+    e.preventDefault();
+    const userName = e.target[0].value;
+    const email = e.target[1].value;
+    const password = e.target[2].value;
+    const newUser = await auth.createUserWithEmailAndPassword(email, password);
+    const createNewSubCollection = await db
+      .collection("users")
+      .doc(newUser.user.uid)
+      .collection("liked")
+      .add({
+        liked: "0",
+      });
+    const setNewUserData = await db
+      .collection("users")
+      .doc(newUser.user.uid)
+      .set({
+        name: userName,
+        isAdmin: false,
+      });
+  };
+  return (
+    <div>
+      <h5>Sign Up</h5>
+      <form onSubmit={(e) => createNewUser(e)}>
+        <input type="text" id="nameField" placeholder="Name" />
```

Comment:
remove these ids, generally we never use ids and they're not used anywhere anyway

Reply:
same with all other ids, use classes instead if you need to